src/pages/thank_you.js

```diff
@@ -1,33 +1,25 @@
-import React from "react";
-
-import Helmet from "react-helmet";
-import { withPrefix } from "gatsby";
+import React, { useEffect } from "react";
 import { useSearchParams } from "../hooks";
 
 import Layout from "../components/Layout";
 import BoxWithLogo from "../components/shared/BoxWithLogo";
 import Seo from "../components/seo";
 
-// scripts provided by Google Ad Words Team
-const GTag = ({ adsTracking }) =>
-  true ? (
-    <Helmet>
-      <script>
-        {typeof window !== "undefined" &&
-          window.gtag("event", "conversion", {
-            send_to: "AW-943271359/NoDFCMXbkuEBEL_b5MED",
-            ...adsTracking,
-          })}
-      </script>
-    </Helmet>
-  ) : null;
-
 const ThankYou = _ => {
-  const adsTracking = useSearchParams();
+  const priceInfo = useSearchParams();
```

Comment:
I think this name change is confusing

Reply:
I thought it made more sense since it's what we're passing into the Google Adwords tracking. 🤷‍♂️

While I was reading it, it just didn't seem accurate to say we were passing adTracking into the Adwords tracking. I can change if if you feel strongly.

Reply:
Never mind, I'll change it back. I forgot that's what we called it in the backend!